test/templates/test_fortified_village-artifacts.wgt

```diff
@@ -0,0 +1,59 @@
+[global]
+map="test/templates/artifacts.wmf"
+multiplayer="true"
+interactive_player="0"
+
+win_condition="artifacts.lua"
+
+# Artifacts is not time based, but we can use this in the script for the timeout.
+win_condition_duration="3"
+
+peaceful="false"
+fogless="false"
+custom_starting_positions="false"
+
+[player_1]
+tribe="amazons"
+init="fortified_village.lua"
+team="0"
+ai="normal"
+closed="false"
+playercolor="0 255 255"
```

Comment:
Why is this set? Could be checked in test.

Reply:
It's set to test the parser code. :) But also testing it is a good idea. (Do we have lua interface for it?)

Reply:
No, we don't. The current lua interface can only report the default playercolour.